config/application.rb

```diff
@@ -67,5 +67,8 @@ class Application < Rails::Application
     # Version info
     config.application_name = "waste-exemptions-back-office-ta"
     config.git_repository_url = "https://github.com/DEFRA/#{config.application_name}"
+
+    # Change authomatic expire of renew's magic link token
```

Comment:
Not sure this comment is applicable here. I think it would be better to use the same comment as in `.env.example` as the basis for one here.

```suggestion
    # Grace window value in days from when a registration expires. During the
    # grace window users will still be permitted to renew their registration.
```

Reply:
Yes! this was an unfortunate copy-paste :D